### ISSUE TITLE
Extra field in backend search

### ISSUE THREAD

**Issue by ugomatic** [TheRestartProject/RepairDirectory#59]
In admin backend, search should include the "Local area / borough" field. This will make it much easier to review data on a borough basis and remove the need to use the address field for the borough.

This means that a "Local area/borough" field should appear in the homepage in the admin backend, so that search includes it - correct @ngm ?

**Comment by ngm** [comment]
Correct, it's pretty straightforward to add in too.  Though the more we add the more cluttered it'll look, so we'll need to think about that.

Do we also want city?
